.jenkins/lsu/env-gcc-12-cuda-12-dgx.sh

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022-2023 Hartmut Kaiser
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+module purge
+module load cmake
+module load gcc/12
+module load cuda/12
+module load boost/1.81.0-${build_type,,}
+module load hwloc
+
+export CXX_STD="17"
+
+configure_extra_options+=" -DHPX_WITH_CXX_STANDARD=${CXX_STD}"
+configure_extra_options+=" -DHPX_WITH_MALLOC=system"
+configure_extra_options+=" -DHPX_WITH_FETCH_ASIO=ON"
+configure_extra_options+=" -DHPX_WITH_COMPILER_WARNINGS=ON"
+configure_extra_options+=" -DHPX_WITH_COMPILER_WARNINGS_AS_ERRORS=OFF"
+configure_extra_options+=" -DHPX_WITH_CUDA=ON"
+configure_extra_options+=" -DHPX_WITH_NETWORKING=OFF"
+configure_extra_options+=" -DHPX_WITH_DISTRIBUTED_RUNTIME=OFF"
+configure_extra_options+=" -DHPX_WITH_ASYNC_MPI=OFF"
+configure_extra_options+=" -DCMAKE_CUDA_ARCHITECTURES='52'"
```

Comment:
```suggestion
configure_extra_options+=" -DCMAKE_CUDA_ARCHITECTURES='80'"
```
I think this should be 80 as the node only contains A100 GPUs.